admittance_controller/src/admittance_rule.cpp

```diff
@@ -240,22 +249,25 @@ controller_interface::return_type AdmittanceRule::update(
     }
   }
 
+  // TODO: Add here desired vector at the IK tip and not tool!!!!
+
   // Do clean conversion to the goal pose using transform and not messing with Euler angles
   convert_array_to_message(relative_desired_pose_vec_, relative_desired_pose_);
   tf2::doTransform(current_pose_control_frame_, desired_pose_, relative_desired_pose_);
 
   // Use Jacobian-based IK
   // Calculate desired Cartesian displacement of the robot
```

Comment:
Update the comment (prob my mistake):

// Convert desired position deltas to joint deltas